lib/maze/client/selenium/bs_browsers.yml

```diff
@@ -138,6 +138,13 @@ ios_16:
   osVersion: "16"
   realMobile: true
 
+ios_17:
+  browserName: "iphone"
```

Comment:
According to https://www.browserstack.com/automate/capabilities, `browserName` doesn't seem to be needed.